reference-implementations/AppGW-IAPIM-Func/bicep/apim/apim.bicep

```diff
@@ -36,7 +38,7 @@ param appInsightsInstrumentationKey string
  * Resources
 */
 
-resource apimName_resource 'Microsoft.ApiManagement/service@2020-12-01' = {
+resource apimName_resource 'Microsoft.ApiManagement/service@2023-05-01-preview' = {
```

Comment:
Instead of preview version, should 2022-08-01 version be used? (https://learn.microsoft.com/en-us/azure/templates/microsoft.apimanagement/change-log/summary#2022-08-01)